plugins/opds.koplugin/opdsbrowser.lua

```diff
@@ -804,12 +856,16 @@ function OPDSBrowser:showDownloads(item)
             end
         end
     end
-    if (#type_buttons % 2 == 1) then -- we need even number of type buttons
-        table.insert(type_buttons, {text = ""})
-    end
-    for i = 2, #type_buttons, 2 do
-        table.insert(buttons, {type_buttons[i - 1], type_buttons[i]}) -- type buttons, two in a row
+    -- handles adding download button(s)
+    if (download_buttons ~= {}) then
+        if (#download_buttons == 1) then
+            table.insert(buttons, download_buttons)
+            table.insert(buttons, {})  -- add separator for stream buttons
```

Comment:
Do you need a separator when there are more than 1 download button?

Reply:
Okay, what if we have one download button and no stream?

Reply:
I left a comment about what I could do on the other review you left.